test/Microsoft.AspNet.Mvc.FunctionalTests/ModelMetadataAttributeTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.TestHost;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class ModelMetadataAttributeTests
+    {
+        private readonly IServiceProvider _services = TestHelper.CreateServices(nameof(ModelBindingWebSite));
+        private readonly Action<IApplicationBuilder> _app = new ModelBindingWebSite.Startup().Configure;
```

Comment:
suggest creating a new `ValidationWebSite` since validation relates to model binding but is not the same sub-system.  the tests for `TryValidateModel` would also fit in the new `ValidationWebSite`.